lib/diagnostic.rb

```diff
@@ -35,7 +45,13 @@
 # "Somerville". Finally, assign the modified Person to `Response.dave` below.
 
 ## replace nil with your response, then continue your work on the next line
-dave = nil
+
+# dave = Hash.new
+dave = Person.new
+# dave = Hash.new('')
+dave = Person.new('')
+dave = {name: "Dave", age: 32, location: "Ohio"}
+
 Response.dave = dave
 ##
 
```

Comment:
Remember that when instantiating a new instance of an object we pass arguments to the .new method
How would you change a property?